python/sidecar/config.py

```diff
@@ -18,9 +18,7 @@
 logging.root.setLevel(logging.NOTSET)
 
 logger = logging.getLogger()
-handler = logging.FileHandler(
-    os.path.join(os.environ.get("SIDECAR_LOG_DIR", "/tmp"), "refstudio-sidecar.log")
-)
+handler = logging.StreamHandler()
```

Comment:
We spoke about keeping the file logging. Can we have that?

Reply:
@cguedes Is your thought to also include the FastAPI server logs in the file?

Reply:
No. Not for now. This is ok 👍 with the current logs being written to the file and and to the console.